lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +21,19 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+
+    only_letters_text = ''
+    for letter in text.lower():
+        if letter.isalpha() or letter == ' ':
+            only_letters_text += letter
```

Comment:
try not to create a new string to save letters from the text, but use `replace` in current text

Reply:
done!